lib/src/container/encapsulate.rs

```diff
@@ -283,8 +283,8 @@ async fn build_impl(
 /// Options controlling commit export into OCI
 #[derive(Debug, Default)]
 pub struct ExportOpts {
-    /// If true, perform gzip compression of the tar layers.
-    pub compress: bool,
+    /// If false, do not perform gzip compression of the tar layers.
+    pub skip_compression: bool,
```

Comment:
This looks odd. Shouldn't it be "If false, perform gzip compression of the tar layers" ? (Or maybe reword as "If true, skip gzip compression of the tar layers".) Another way to see this I think is we added one negation in the field name, but two negations in the docstring.